tests/Directory.Build.props

```diff
@@ -8,16 +8,10 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Update="coverlet.collector">
-            <PrivateAssets>all</PrivateAssets>
-            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-        </PackageReference>
+        <PackageReference Include="coverlet.collector" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" />
         <PackageReference Include="xunit" />
-        <PackageReference Update="xunit.runner.visualstudio">
-            <PrivateAssets>all</PrivateAssets>
```

Comment:
Using `PrivateAssets` here (which I think I added when I switched to `Directory.Build.props`) was preventing test discovery so `dotnet test` was "passing" with `0` discovered tests.  🤦‍♂️